app/controllers/admin/site_settings_controller.rb

```diff
@@ -33,7 +33,7 @@ def update
 
     case SiteSetting.type_supervisor.get_type(id)
     when :integer
-      value = value.gsub(/\D/, "")
+      value = value.tr("^-0-9", "")
```

Comment:
This is not perfect, as you can still input e.g. `-1-1`, but I think the expedience of this fix is more important than that edge case.

Reply:
Yep I agree, also this will all become a moot point anyway when we change this to be a native Number input :+1: